src/components/manage/Controlpanels/Controlpanel.jsx

```diff
@@ -19,6 +19,7 @@ import { updateControlpanel, getControlpanel } from '@plone/volto/actions';
 
 import saveSVG from '@plone/volto/icons/save.svg';
 import clearSVG from '@plone/volto/icons/clear.svg';
+import { rest } from 'lodash';
```

Comment:
@danalvrz This is no longer used, so should be removed

Reply:
(If you look at the checks at the bottom of the PR, you'll see that the eslint check failed because of this)